controller/contacts.js

```diff
@@ -0,0 +1,50 @@
+const {HttpError, controlWrapper} = require('../helpers');
+const contacts = require('../models/contacts');
+
+const getContactsAll = async (req, res) => {
+  const result = await contacts.listContacts();
+  res.json(result);
+};
+
+const getContactById = async (req, res) => {
+  const {contactId} = req.params;
+  const result = await contacts.getContactById(contactId);
+
+  if(!result) throw HttpError(404, "Not found");
```

Comment:
```suggestion
  if(!result){
   throw HttpError(404, "Not found");
   }
```
використовуйте фігурні дужки після if statement